.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+<!--
+We appreciate your contribution to the OpenTelemetry project! 👋🎉
+
+Before creating a pull request, please make sure:
+- Your PR is solving one problem
+- Please provide enough information so that others can review your pull request
+- You have read the guide for contributing
+  - See https://github.com/open-telemetry/opentelemetry-js/blob/master/CONTRIBUTING.md
+- You signed all your commits (otherwise we won't be able to merge the PR)
+  - See https://github.com/open-telemetry/community/blob/master/CONTRIBUTING.md#sign-the-cla
+- You added unit tests for the new functionality
+- You mention in the PR description which issue it is addressing, e.g. "Fixes #xxx". This will auto-close
+  the issue that your PR fixes (if such)
+-->
+
+## Which problem is this PR solving?
+
+- Fixes #
```

Comment:
Since it wouldn't always be the case that a PR is closing an existing issue, I would leave that out. It's also semantically incorrect for features which are "closed" and not "fixed".

Reply:
Ok, done.